packages/ui-app/package.json

```diff
@@ -18,6 +18,7 @@
     "@polkadot/ui-settings": "^0.28.8",
     "@polkadot/ui-util": "^0.28.8",
     "@types/chart.js": "^2.7.45",
+    "@types/classnames": "^2.2.7",
```

Comment:
Could just use https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/util/classes.ts as is done everywhere.

Reply:
Totally! I didn't see that. I got confused because I saw the classnames module in the node_modules folder so I just assumed that it's used elsewhere.  Probably just as a dependency by another module